cmd/buildx/main.go

```diff
@@ -21,6 +21,10 @@ import (
 	_ "k8s.io/client-go/plugin/pkg/client/auth/oidc"
 	_ "k8s.io/client-go/plugin/pkg/client/auth/openstack"
 
+	_ "github.com/moby/buildkit/client/connhelper/dockercontainer"
```

Comment:
These should probably go to remote driver pkg. The driver expects them to be always loaded.